content/english/episodes/ep-8-trailer.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Trailer: Episode 8 with Mojdeh Rastgoo"
+meta_title: "Trailer: Episode 8 with Mojdeh Rastgoo"
+description: "A preview of our chat with Mojdeh Rastgoo. Watch the full episode on April 15, 2025"
+date: 2025-04-10T00:00:00-08:00
+host: ["Tereza", "Georgi"]
+draft: false
+tags: ["python", "iran", "france", "pyladies", "psf-coc-wg", "pypodcats", "community", "project manager"]
+publish_date: 2025-04-10T00:00:00-08:00
+podcast_file: "/audio/hidden-figures-ep8-trailer.mp3"
+podcast_duration: 98
+episode_image: "images/ep8-mojdeh.png"
+podcast_bytes: "3889257" # the length of the episode in bytes
+images: ["images/ep8-mojdeh.png", "images/mojdeh.jpg"]
+explicit: false 
+type: 'episode'
+episode: '8'
+season: 1
+transcript: '/transcript/hidden-figures-of-python-ep8-trailer.srt'
+episode_type: trailer
+layout: episode
+youtube_id: "qetbNfnCkJM"
+  
+speaker:
+  url: "/speakers/mojdeh-rastgoo/"
+  title: "Mojdeh Rastgoo"
+  image: "/images/mojdeh_rastgoo.jpeg"
+  description: "
+Mojdeh Rastgoo is a Machine Learning Data Scientist and Project Manager based in Pris.
+She has a PhD in computer sicence and has been an active memnber of the community since 2018. 
```

Comment:
```suggestion
She has a PhD in Computer Science and has been an active member of the community since 2018. 
```

Reply:
"of the community", should it say Python community, or open source community, perhaps?